data/governments.txt

```diff
@@ -151,6 +151,20 @@ government "Korath"
 	
 	"player reputation" -10
 
+government "Korath (Berserk)"
+	swizzle 0
+	color .8 .5 .1
+	language "Korath"
+	
+	"attitude toward"
+		"Kor Sestor" -1
```

Comment:
True. At least for the foreseeable future, anyway.

Reply:
One thing I'd like to note: There's no guarantee that the player has completed any major plot line before starting these missions. All they need to find the Remnant is one of:
a) Key stone from the Hai, or
b) Acquire a Jump Drive from a Korath raiding the edge of Syndicate space.

They do need to get to Hai space to be able to finish the initial missions, though, since they're the only place to acquire 50 keystones.

Reply:
(oh, and I made the change you suggested)